src/styles/blocks/card.scss

```diff
@@ -0,0 +1,93 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $mainColor;
+  border-radius: 5px;
+  transition: $transitionForAnimation;
```

Comment:
For `transition`, it is also worth specifying what exactly to use it for.
https://developer.mozilla.org/en-US/docs/Web/CSS/transition-property